src/cli/format.ts

```diff
@@ -5,20 +5,11 @@ import { createLogger, log } from '../utils/logging';
 
 import { runESLint } from './adapter/eslint';
 import { runPrettier } from './adapter/prettier';
-import { tryAddEmptyExports } from './configure/addEmptyExports';
-import { tryPatchDockerfile } from './configure/patchDockerfile';
-import { tryPatchRenovateConfig } from './configure/patchRenovateConfig';
-import { tryPatchServerListener } from './configure/patchServerListener';
 import { tryRefreshIgnoreFiles } from './configure/refreshIgnoreFiles';
```

Comment:
Is it worth modelling this as an "evergreen upgrade", so the commands only have to worry about invoking `upgradeSkuba` and we could introduce other upgrades of its nature in future?

Reply:
Hmmm I'll leave it for now 🤔